frb_codegen/src/ir/ty.rs

```diff
@@ -69,21 +79,21 @@ pub trait IrTypeTrait {
 
     fn dart_api_type(&self) -> String;
 
-    fn dart_wire_type(&self) -> String;
+    fn dart_wire_type(&self, wasm: bool) -> String;
```

Comment:
(Just as a complement to comments above) if we make it something like `target: GenerationTarget` where `enum GenerationTarge {normal, wasm}` or sth like that, we have better typeing system - a `bool` positional argument can be misused, while nobody will pass in wrong values to a  `GenerationTarget`-typed argument